docs/docs/miscellaneous/importing-exporting-datasources.mdx

```diff
@@ -61,15 +61,39 @@ superset export_datasource_schema
 
 As a reminder, you can use the `-b` flag to include back references.
 
-### Importing Datasources from YAML
+### Importing Datasources
 
-In order to import datasources from a YAML file(s), run:
+In order to import datasources from a ZIP file, run:
+
+```
+superset import_datasources -p <path / filename>
+```
+
+The optional username flag **-u** sets the user used for the datasource import. The default is 'admin'. Example:
+
+```
+superset import_datasources -p <path / filename> -u 'admin'
+```
+
+### Legacy Importing Datasources
+
+#### From older versions of Superset to current version
+
+When using Superset version 4.x.x to import from an older version (2.x.x or 3.x.x) importing is supported as the command `legacy_import_datasources` and expects a JSON or directory of JSONs. The options are `-r` for recursive and `-u` for specifying a user. Example:
```

Comment:
You say the options are `-r` and `-u` but then the example has `-p` -- is that correct?  Just checking that `-p` is supported here and that you don't want to demo `-r` or `-u` in the command.

Reply:
Hello sfirke, the command with -p is correct as that is a required parameter, but it is confusing in this example to have not included either the optionals. I think clarifying this will help.